### DISCUSSION TITLE
is_current not working for redirect page under parent.

### DISCUSSION THREAD

**Discussion by niloc7** [statamic/cms#4723] [Q&A]
I'm using this code for my links within a product page:

`class="{{ if is_current }} font-semibold` {{ endif }}`

It works fine for all my links which are basically child pages inside a page except for the first page because the first page is a URL redirect - to the parent page.

Any ideas on how to solve this?

Parent Page
- Child Page 1 (which redirects to the Parent Page)
- Child Page 2
- Child Page 3
- Child Page 4
- Child Page 5

So it only works when I am in Pages 2-5.

**Comment by jackmcdade** [comment]
So you want to say you're on Child Page 1 even though you're on Parent?

**Reply by niloc7** [reply]
Sort of, cause Child Page 1 is "Introduction".

I did it this way, because all the other pages require a reusable info from the parent page (ie. product specialty, product code).

**Reply by jackmcdade** [reply]
Okay gotcha. Maybe something like this would work? You might need to adjust based on the depth.

```
{{ if (is_parent && ! segment_2) || is_current }} font-semibold {{ /if }}
```

**Reply by niloc7** [reply]
It doesn't seem to be working.. no errors, but just doesn't work. 

I suspect, maybe on that First Child page, the parent would be different - meaning it's parent would be Home.

I tried this as well, but it comes out with an error:

`{{ if is_current || page:slug == segment_1 }} font-semibold {{ endif }}`

**Reply by niloc7** [reply]
`Statamic\Modifiers\ModifierNotFoundException`
`Modifier [] not found`

This is the error code btw. And I suspect it's because of using a colon : inside it. But how else would we get page:slug inside hmm...

**Reply by niloc7** [reply]
Hooray! I found a temporary (or maybe permanent) solution!

I remembered that my Parent Page is the one containing the special data to be used in all the child pages, which is "product-id".

That means only the Parent Page would have that info.

So essentially:

`class="{{ if is_current || product-id }} font-semibold {{ endif }}`

**Comment by niloc7** [comment]
Sort of, cause Child Page 1 is "Introduction".

I did it this way, because all the other pages require a reusable info from the parent page (ie. product specialty, product code).

**Reply by jasonvarga** [reply]
Please reply to the answer itself instead of creating a completely new answer.

**Reply by niloc7** [reply]
Oh man, sorry never knew.

**Reply by jasonvarga** [reply]
No worries, it's not super clear. 👍
